AdyenComponents/BACS Direct Debit/BACSDirectDebitComponent.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol BACSDirectDebitRouterProtocol {
+    func presentConfirmation(with data: BACSDirectDebitData)
+    func confirmPayment(with data: BACSDirectDebitData)
+    func cancelPayment()
+}
+
+/// A component that provides a form for BACS Direct Debit payments.
+public final class BACSDirectDebitComponent: PaymentComponent, PresentableComponent {
+
+    // MARK: - PresentableComponent
+
+    public var viewController: UIViewController
+
+    public var requiresModalPresentation: Bool = false
```

Comment:
can you add `/// :nodoc:` to both properties?

Reply:
Added, thanks!